src/modules/aws/infra-cost/infra-cost.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { Logger } from '@aws-lambda-powertools/logger';
+import { InfraCostService } from './infra-cost.service';
+
+@Controller('services-cost')
+export class InfraCostController {
+    private readonly logger = new Logger();
+    constructor(
+        private readonly costService: InfraCostService
+    ) { }
+
+    @Get('infra-cost')
+    async getInfrastructureCost(
+        @Query('start') start: string,
```

Comment:
Can you validate start/end date, per following suggestion:
https://stackoverflow.com/questions/71622866/a-clean-way-to-check-for-query-parameter-in-nestjs